sydent/http/servlets/__init__.py

```diff
@@ -115,16 +116,19 @@ def inner(self, request, *args, **kwargs):
 
 def deferjsonwrap(f):
     def reqDone(resp, request):
+        request.setHeader("Content-Type", "application/json")
         request.setResponseCode(200)
         request.write(json.dumps(resp).encode("UTF-8"))
         request.finish()
 
     def reqErr(failure, request):
```

Comment:
Just add the `request.setHeader("Content-Type", "application/json")` bit outside of the if statement?